utf8proc.sh

```diff
@@ -5,6 +5,10 @@ source: https://github.com/JuliaStrings/utf8proc
 build_requires:
  - "GCC-Toolchain:(?!osx)"
  - alibuild-recipe-tools
+prefer_system: "(?!osx)"
+prefer_system_check: \
+  brew --prefix utf8proc > /dev/null;
+  if [ $? -ne 0 ]; then printf "Use brew install utf8proc"; exit 1; fi
```

Comment:
I would have a test which compiles utf8proc.h, like done elsewhere... Something along the lines of:

```bash
printf "#include <utf8proc.h>\n" | c++ -c -I$(brew --prefix utf8proc)/include -xc++ - -o /dev/null 2>&1
```